### DISCUSSION TITLE
Connecting to Meeting Guide App with Google Sheet

### DISCUSSION THREAD

**Discussion by Hiro-Tokyo** [code4recovery/12-step-meeting-list#1259] [Q&A] [answered]
Hello team!

Kansai Central Office is attempting to connect Meeting Guide App. Meeting Guide App support says they have some issues to import the data.

_We attempted to load the data feed provided but received mostly errors - please see attached excel file.

It looks like the google sheet needs some adjustments, the 2 primary ones are:
Change column header "slug" to "ID"
Provide a day in the format in this [google sheet template ](https://docs.google.com/spreadsheets/d/1iA8oVtddHVEZ8gslWPlTrfBfbgJpiS0Tt6sFOTi_5dk/edit#gid=687617754) (that is the error you will see in the spreadsheet attached)

Regarding the "day" column - the issue may be a translation issue when we import it - can you confirm that what is in the column should translate to English as "Sunday", "Monday" etc.?

Also, it looks like your website is using the TSML plug-in- which generates a json string which is usually easier to import than google sheets - can you send in the json instead?  Then the "day" field will be numeric which should help._

Website meeting list URL:
https://www.aa-kco.org/meetings/

Google Sheet URL:
https://docs.google.com/spreadsheets/d/1JSqXehcLk91qffMHRIx-8ddryYEC-vnrqb6aGHmApys/edit?usp=sharing

1. We can replace "slug" with "ID" easier. Will it be good way?
2. How we can generate a json string of meeting list Google Sheet?
 _TSML plug-in- which generates a json string_

Please help support us for these issue.

**Comment by joshreisner** [comment, marked as answer]
hi hiro,

you should be fine to change Slug to ID, TSML UI should still work

it's too bad the meeting guide importer can't use the japanese day names, but TSML UI can handle those as well. `Sunday`, `Monday`, etc. this won't impact what TSML UI displays

not sure why they are confused and think you are using the plugin, but tell them you are not. (there is a way to convert to JSON but this is an extra step that you will have to do every time there is a change so I don't recommend this because it is liable to get out of sync)

**Comment by Hiro-Tokyo** [comment]
Thanks Josh! We replaced slug with ID, Japanese day names with English day names. We will contact the app support!
